tools/reckless

```diff
@@ -0,0 +1,684 @@
+#!/usr/bin/env python3
+
+from subprocess import Popen, PIPE
+import sys
+import json
+import os
+import argparse
+from pathlib import Path
+import shutil
+import tempfile
+import requests
+
+
+repos = ['https://github.com/lightningd/plugins']
+
+
+def py_entry_guesses(name):
+    return [name, f'{name}.py', '__init__.py']
+
+
+def unsupported_entry(name):
+    return [f'{name}.go', f'{name}.sh']
+
+
+class InstInfo:
+    def __init__(self, name, url, git_url):
+        self.name = name
+        self.repo = url         # Used for 'git clone'
+        self.git_url = git_url  # API access for github repos
+        self.entry = None
+        self.deps = None
+        self.subdir = None
+        self.commit = None
+
+    def __repr__(self):
+        return f'<InstInfo object>\n'\
+               f'name: {self.name}\nrepo: {self.repo}\ngit: {self.git_url}'\
+               f'\nentry:{self.entry}\ndepency source:{self.deps}'
```

Comment:
```suggestion
               f'\nentry:{self.entry}\ndependency source:{self.deps}'
```

Reply:
Multiline `__repr__`s are a bit strange, but you can easily achieve multi-line strings this way:

```
f"""<InstInfo object>
name: {self.name}
repo: {self.repo}
git: {self.git_url}
entry:{self.entry}
dependency source:{self.deps}"""
```
That way we can read the output in the code as it would appear in the output. If indentation is an issue then `textwrap.dedent` might be useful.